### DISCUSSION TITLE
ETH/TON DUAL MINING FIX for LHR tuning loop not yet mentioned?

### DISCUSSION THREAD

**Discussion by Burchoid** [Lolliedieb/lolMiner-releases#1228] [General]
Win 10
3060 TI LHR

I have been fighting a LHR lock loop for the last several days where the program would essentially start and and pause for hours on end without actually tuning the card.  It got to LHR tune -1500 when I decided to try letting it run overnight.  Once stuck in that loop, the LHR tune doesn't seem to be actually doing anything.

Drivers did not change anything.

Locking the CLK did not change anything.

Running as ADMIN did not change anything.

**What finally worked was disabling G-SYNC in the Nvidia control panel.**   I can now repeatedly restart and re-tune without a loop, and it seems none of the other "fixes" listed on the numerous closed issues did anything for me.  I am currently using the prior driver (not 500 series) and not running anything as admin, and I am using a monitor with this computer just fine.  

Lastly, it seem odd that even the act of OPENING the nvidia control panel triggers a "lock detected" message in lolminer. 

Figured I ought to share these details in case it helps the developer or another user.

Thanks for the great program!!

**Comment by UserDC-LeGrand** [comment]
I am dual-mining Eth/TonCoin with lolMiner v1.44 on three HiveOS rigs with a total of 30 Nvidia GPUS series 1xxx, 2xxx, and 3xxx. It took a while to stabilize everything, including power, heat management, PSU cable upgrades, and a flaky GPU.

Did not run into LHR tuning loop you mentioned on any of my LHR V2 cards, including my 3060ti. Have you tried the 5x lolMiner kernel variations (--mode a, b , LHR2,  and LHRLP) pointed at your problem GPU, as this could have an impact.

Here are my 3060ti dual-mining and very stable:

![image](https://user-images.githubusercontent.com/99490445/153688027-f0006e87-3085-4bae-aa34-d40a92a97f17.png)

![image](https://user-images.githubusercontent.com/99490445/153687984-3bc04901-4960-45aa-a9b0-f9a5466ed752.png)

I found an article that stated that a specific NVIDIA driver version for Linux would provide greatest stability (470.74), so I went with that first. 

I also noticed that the --lhrwait option should be pushed out to until your OC setting have kicked in, so I set that parameter to 60 seconds, and my OC delay setting to 15 seconds.

There is also an option for --dagdelay which can stabilize your miner by spreading out the initial DAG builds, so you may want to look into that.

I had to force a very low --cclk setting on one of my fragile 2070Super GPU, so that the miner on that rig would not crash on startup and initial dag building. The great thing with lolMiner is that it raises back the Core clock right back when the miner is done tuning everything. So my 2070 Super which other miners could not handle. I ws about to give up and throw away my 2nd hand 2070Super GPU, but lolMiner saved the day!!!

Very impressed by what this miner can accomplish, its auto-tuning, its healing options, etc. This is after trying other leading miners (T-rex, NBMiber, Gminer)...

Hope that some of this helps you in your troubleshooting. 

Good luck!

**Comment by nguyenhathienphu** [comment]
> 

same problem,
eternal loop, 
![Untitled](https://user-images.githubusercontent.com/90582202/153690470-57b4d978-d334-4572-b185-b1984e85444c.jpg)

**Comment by DaethStaR** [comment]
Hello. Here is the thing. Not all of us have big farms or use HiveOS. I have two gpus on two different computers. My 2080 mines fine my 3070ti gets suck in the Unlocking state multiple times and hour. When it is running correctly, I get 50ish MH on Eth and 2.2Gh on Ton. Unfortunately, with the continuous unlocking, it really slows down the mining. I have used T-Rex dual mining and do not have this issue with Eth/ALTH but I do not want to mine Alethium. Hopefully someone can figure out if it is parameter issue where we need to enter something or if it is a software issue with certain LHR cards in WINDOWS where this issue seems to be happening.
